### PR TITLE
feat : Completed login page for my e-com Blossom store

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Screenshot (354)
Completed login page for my e-com Blossom store

### CODE REVIEW

**Review by FarhanMobashir — APPROVED** [review]
code is clean . can be merged

File: Authentication/login-page.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+nav{
```
Comment (vidhi-thakur) [inline Authentication/login-page.css:1]: could use class name here

File: Authentication/login-page.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+.main-div{
+    height: 100vh;
+    background-image: url("/assets/login-page-bg-img.jfif");
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.form-inputs{
+    background-color: var(--white);
+    border: 2.5px solid var(--black);
+    box-shadow: 5px 0px var(--gray);
```
Comment (vidhi-thakur) [inline Authentication/login-page.css:13]: the value of box shadow could be stored in a css variable

File: Authentication/login-page.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                Password
+                   <input type="password" name="password" class="input" placeholder="* * * * * * * *">
+               </label>
+                
+               <div class="forgot-pass-div flex">
+                   <label>
+                    <input type="checkbox" /> Remember me
+                   </label>
+<p class="forgot-pass">
+    <a href="/" class="forgot-pass-link" >
```
Comment (vidhi-thakur) [inline Authentication/login-page.html:32]: you can use `Prettier` for code formatting

**Review by vidhi-thakur — APPROVED** [review]
suggested small changes

**Pull request merged**